src/conn/conn_api.c

```diff
@@ -2667,6 +2667,34 @@ __conn_write_base_config(WT_SESSION_IMPL *session, const char *cfg[])
     return (ret);
 }
 
+/*
+ * __conn_set_key_management --
+ *     Configure a custom key management implementation on database open.
+ */
+static int
+__conn_set_key_management(
+  WT_CONNECTION *wt_conn, WT_KEY_MANAGEMENT *key_management, const char *config)
+{
+    WT_CONNECTION_IMPL *conn;
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session;
+
+    conn = (WT_CONNECTION_IMPL *)wt_conn;
+    CONNECTION_API_CALL(conn, session, set_key_management, config, cfg);
+    WT_UNUSED(cfg);
+
```

Comment:
If we're not using config, we should explicitly check that is either NULL or strlen() == 0.